vendor/github.com/jesseduffield/gocui/gui.go

```diff
@@ -1152,6 +1152,23 @@ func (g *Gui) flush() error {
 	return nil
 }
 
+// force redawing a view outside of the normal main loop. Useful during longer
+// operations that block the main thread, to update a spinner in a status view.
+func (g *Gui) ForceRedrawView(v *View) error {
+	for _, m := range g.managers {
+		if err := m.Layout(g); err != nil {
+			return err
+		}
+	}
```

Comment:
We layout everything here for laziness; it would be enough to just set the size of the view that we're redrawing (the AppStatus view in this case), but I haven't looked much into how we would achieve that.